routes/web.php

```diff
@@ -50,9 +50,9 @@
 */
 Route::get('/controllers/roster', 'RosterController@index');
 Route::get('/controllers/staff', 'RosterController@staffIndex');
-Route::get('/login', 'RosterController@login');
-Route::get('login', [ 'as' => 'login', 'uses' => 'RosterController@login']);
-Route::get('/logout', 'RosterController@logout');
+// VATSIM Connect SSO
+Route::get('/login', 'Auth\LoginController@login')->middleware('guest')->name('login');   // Login
+Route::get('/logout', 'Auth\LoginController@logout')->middleware('auth')->name('logout'); // Logout
```

Comment:
These comments here aren't necessary and are redundant since the code here is fairly self explanatory.

Reply:
Will remove and cleanup